conanfile.py

```diff
@@ -0,0 +1,33 @@
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
+
+
+class MeshlinklibConan(ConanFile):
+    name = "meshlink"
+    version = "0.1"
+    license = "<Put the package license here>"
+    author = "<Put your name here> <And your email here>"
+    url = "<Package recipe repository url here, for issues about the package>"
+    description = "<Description of Elearcommonlib here>"
+    topics = ("<Put some tag here>", "<here>", "<and here>")
+    settings = "os", "compiler", "build_type", "arch"
+    #requires="catta/0.1@demo/dev"
+    options = {"shared": [True, False]}
+    default_options = "shared=False"
+    generators = "make"
+
+
+    def build(self):
+        autotools = AutoToolsBuildEnvironment(self)
+        self.run("cd .. && autoreconf -fsi ")
+        autotools.configure(configure_dir="..",args=["--prefix=${PWD}"])
+        autotools.make()
+        autotools.install()
+    def package(self):
+        self.copy("*.h", dst="include", src="src/include/")
+        self.copy("*.h", dst="include", src="catta/include/include")
+        self.copy("*", dst="lib", src="src/lib", keep_path=False)
+        self.copy("*", dst="lib", src="catta/lib", keep_path=False)
```

Comment:
Does this also copy header files?

Reply:
yes, It copies headers in include directory of package, .so & .a to lib directory